spec/helpers/comments_helper_spec.rb

```diff
@@ -60,5 +60,24 @@
         expect(helper.css_classes_for_comment(unreviewed_comment)).to eq("unreviewed comment group user-#{user.id}")
       end
     end
+
+    context "when comment author is an official" do
+      let(:official_user) { create(:official) }
+      let(:comment) { create(:comment, pseud: official_user.default_pseud) }
+  
+      it "has official class" do
+        expect(helper.css_classes_for_comment(comment)).to match("official")
+      end
+
+      context "comment author is also creator of a work inside anonymous collection" do
+        let(:anonymous_collection) { create(:anonymous_collection) }
+        let(:work) { create(:work, authors: [official_user.default_pseud], collections: [anonymous_collection]) }
```

Comment:
Since we're not referencing the anon collection elsewhere, we could just do

```
let(:work) { create(:work, authors: [official_user.default_pseud], collections: [create(:anonymous_collection)]) }
```